File: 프로그래머스/lv1/예산.py
def solution(d, budget):
    d.sort()
    total=0
    answer=0
    for i in range(len(d)):
        total+=d[i]
        if total<=budget:
            answer+=1
        else:
            break
    return answer

print(solution([1,3,2,5,4],9))

File: 프로그래머스/lv1/예산.java
package lv1;

import java.util.Arrays;

public class 예산 {
    public int solution(int[] d, int budget) {
        int answer = 0;
        Arrays.sort(d);
        for(int money:d){
            budget-=money;
            if(budget<0){
                break;
            }
            answer++;
        }
        return answer;
    }

    public static void main(String[] args) {
        예산 price = new 예산();
        System.out.println(price.solution(new int[]{1,3,2,5,4},9));
    }
}
